crates/atuin/src/shell/atuin.zsh

```diff
@@ -27,6 +27,12 @@ fi
 export ATUIN_SESSION=$(atuin uuid)
 ATUIN_HISTORY_ID=""
 
+_atuin_update_title() {
+    if [[ "$TERM" == "xterm*" ]]; then
```

Comment:
`"xterm*"` doesn't work. If you quote `*` inside the double quotes `"..."`, it only matches the literal character `'*'`. Here, we expect a wildcard `*` of the glob pattern, which should not be quoted.

```suggestion
    if [[ "$TERM" == xterm* ]]; then
```

Reply:
Ah right, and you did recommend that the first time, sorry about that. Need more coffee...